spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,43 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response!(app_id: nil, platform: nil, review_id: nil, response: "")
```

Comment:
These values must not be `nil`.
I think you can drop argument labels.

```ruby
def create_developer_response!(app_id, platform, review_id, response)
end
```

Labels are needed?